### PR TITLE
Remove: Agents

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy

### CODE REVIEW

File: server/lib/backends/capi/index.js [hunk 1]
```diff
@@ -178,7 +178,8 @@ export default class {
 			this.deps.apiClient.things({
 				identifierValues: uuids,
 				identifierType: type,
-				authority: 'http://api.ft.com/system/FT-TME'
+				authority: 'http://api.ft.com/system/FT-TME',
+				agent: undefined
```
Comment (ironsidevsquincy) [inline server/lib/backends/capi/index.js]: does `things` take an agent?

File: server/lib/backends/capi/index.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class {
 		metrics.count(`backends.${this.type}.page.count`, 1);
 		const cacheKey = `${this.type}.page.${uuid}`;
 		const fetcher = () =>
-			this.deps.apiClient.pages({ uuid })
+			this.deps.apiClient.pages({ uuid, agent: undefined })
```
Comment (ironsidevsquincy) [inline server/lib/backends/capi/index.js]: does `pages` take an agent?

File: server/lib/backends/fast-ft.js [hunk 3]
```diff
@@ -24,7 +24,8 @@ export default class {
 					}
 				},
 				offset: from,
-				count: limit
+				count: limit,
+				agent: undefined
```
Comment (ironsidevsquincy) [inline server/lib/backends/fast-ft.js]: can just leave this off (same as undefined)

File: server/lib/backends/capi/get-search-opts.js [hunk 4]
```diff
@@ -75,6 +75,7 @@ export default function (termName, termValue, { since, type, from = 0, limit = 1
 			}
 		});
 	}
+	searchOpts.agent = undefined;
```
Comment (ironsidevsquincy) [inline server/lib/backends/capi/get-search-opts.js]: maybe keep this in `searchOpts` at the top

File: server/lib/backends/capi/index.js (hunk 2 above)
Comment (andygout) [inline server/lib/backends/capi/index.js]: Yes - will add.

File: server/lib/backends/capi/index.js (hunk 1 above)
Comment (andygout) [inline server/lib/backends/capi/index.js]: Yes - will add.

File: server/lib/backends/capi/get-search-opts.js (hunk 4 above)
Comment (andygout) [inline server/lib/backends/capi/get-search-opts.js]: Ah, yes.

File: server/lib/backends/fast-ft.js (hunk 3 above)
Comment (ironsidevsquincy) [inline server/lib/backends/fast-ft.js]: Sorry, yes, it uses search. Ignore

**Pull request merged**